go.mod

```diff
@@ -3,22 +3,44 @@ module github.com/goreleaser/nfpm
 go 1.14
 
 require (
+	github.com/Djarvur/go-err113 v0.1.0 // indirect
```

Comment:
missing a `go mod tidy`?

Reply:
even after a `make deps` and an explicit `go mod tidy` this still shows up